tests/methods/ParseTest.php

```diff
@@ -137,7 +137,7 @@ public function test_Piwik_data() {
      * @depends testSepRowAutoDetection
      */
     public function testGetColumnDatatypes() {
-        $this->csv->auto('tests/methods/fixtures/datatype.csv');
+        $this->csv->auto(__DIR__ . '/fixtures/datatype.csv');
```

Comment:
There was one other dataprovider which needs that change?

Reply:
I'll remove this change because it is solved with the `chdir()` call in Bootstrap.php. I'll change it tonight at the latest.